README.md

```diff
@@ -107,16 +107,35 @@ The general idea of portals is summed up above: inset pre-rendering, activation,
 
 ### Privacy threat model and restrictions
 
+The Portals design is intended to comply with the [W3C Target Privacy Threat Model](https://w3cping.github.io/privacy-threat-model/). This section discusses the aspects of that threat model that are particularly relevant to Portals and how the design satisfies them.
+
+A portal can contain either a same-site or cross-site resource. Same-site portals don't present any privacy risks, but cross-site resources risk enabling [cross-site recognition](https://w3cping.github.io/privacy-threat-model/#model-cross-site-recognition) by creating a messaging channel across otherwise-partitioned domains. For simplicity, when a cross-site channel needs to be blocked, we also block it for same-site cross-origin portals.
+
+This design assumes that storage will be [partitioned](https://github.com/privacycg/storage-partitioning), even though that is not the status quo in all browsers. Because portaled documents can be activated into a top-level browsing context, they (eventually) live in the first-party storage partition of their origin. This means we have to prevent communication with the surrounding document to the same extent we prevent it with a cross-site link opened in a new tab.
```

Comment:
Are we going to follow the lead of partitioning here, or might we come up with a simpler system like "no storage until activation" rather than partition?

Reply:
I meant this to be compatible with "no storage until activation", as the partitioning explainer anticipates that some storage will be blocked from a third-party context: https://github.com/privacycg/storage-partitioning/issues/8. The "(eventually)" hints at this, but I wonder if you have an idea for being clearer about it, without using lots of words?

Reply:
Not sure I can do fewer words, but maybe:

> This design assumes that storage will be partitioned/blocked in line with the browser's implementation (or lack thereof) of [storage-partitioning](https://github.com/privacycg/storage-partitioning).

Reply:
I've added "or blocked" to try to capture this. How's the new text?